ha_cluster_exporter.go

```diff
@@ -73,15 +73,33 @@ func landingpage(w http.ResponseWriter, r *http.Request) {
 		</html>`))
 }
 
+func loglevel(opt string) {
+	switch opt {
+	case "error":
+		log.SetLevel(log.ErrorLevel)
+	case "warn":
+		log.SetLevel(log.WarnLevel)
+	case "info":
+		log.SetLevel(log.InfoLevel)
+	case "debug":
+		log.SetLevel(log.DebugLevel)
+	}
```

Comment:
we miss here a `default` clause.
something like 
``
default:
		return errors.Errorf("unrecognized log level %q", s)
}
```

Reply:
Thinking of something such as:
```
        default:
                log.SetLevel(log.InfoLevel)
                log.Warnln("Unrecognized log level, default to `info` level")
        }
```

Reply:
This said we can do away with `log.SetLevel(log.InfoLevel)` since default behaviour is to use `info` level